CMSgov/design-system
# [WNMGDS-2341] Fix inverse stories in doc pages

## Summary

Whenever there were both normal and inverse stories on a Storybook component doc page, they would interfere with each other. This is because we were toggling the inverse class imperatively in the story decorators. They were manipulating the same body element. Whatever story came last won. It was chaos. I'm not in love with the fix, but it's at least declarative, and stories won't be interfering with each other.

Before/After:
![Before/after shot of broken and fixed stories](https://github.com/CMSgov/design-system/assets/7595652/d432b581-3032-481e-ab99-32c7c4b37822)


## How to test

View any of the affected component doc pages in storybook and make sure normal (on-light) stories don't have illegible, white labels and that inverse stories are on a dark background.

## Checklist




### If this is a change to code:


### If this is a change to documentation:
